src/pages/BucketList/styles.ts

```diff
@@ -5,13 +5,72 @@ const Container = styled.div`
   width: 100%;
   position: relative;
   display: flex;
+  flex-direction: row;
+  max-width: 1728px;
+  max-height: 1117px;
+  margin: 40px;
+  gap 77px;
+`;
+
+const LeftContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  gap: 100px;
+  width: 100vh;
+  max-width: 300px;
+`;
+
+const RightContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 100vh;
+  max-width: 1367px;
+`;
+
+const Title = styled.h1`
+  font-size: 36px;
+`;
+
+const SideNavigation = styled.div`
+  width: 100%;
+  display: flex;
   flex-direction: column;
 `;
 
-const Title = styled.h1``;
+const DashboardButton = styled.button`
+  width: 100%;
+  height: 75px;
+  font-size: 28px;
+  background-color: #21BA71;
+  border-radius: 18px;
+  color: white;
+  border: none;
+  font-style: normal;
+  font-weight: 700;
+  margin-bottom: 87px;
+`;
+
+const Button = styled.button`
+  width: 100%;
+  height: 35px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 28px;
+  color: #6D6D6D;
+  border: none;
+  background-color: white;
+  margin-bottom: 84px;
+`;
 
-const SideNavigation = styled.div``;
+const ExploreContainer = styled.div`
+  width: 100vh;
```

Comment:
I would use view width here
```suggestion
  width: 65vw;
```